app/views/pageflow/entries/_meta_tags.html.erb

```diff
@@ -0,0 +1,4 @@
+  <meta name="dc.language" content="<%= I18n.locale %>" />
+  <% if entry.keywords.present? %><meta name="keywords" content="<%= entry.keywords %>"><% end %>
+  <% if entry.author.present? %><meta name="author" content="<%= entry.author %>"><% end %>
+  <% if entry.publisher.present? %><meta name="publisher" content="<%= entry.publisher %>"><% end %>
```

Comment:
Let's also add a test this logic with fallback to config settings works. It can be a [controller test for `EntriesController#show`](https://github.com/codevise/pageflow/blob/master/spec/controllers/pageflow/entries_controller_spec.rb#L164). That spec is already a bit longer than I'm comfortable with, but I guess the cases covered there are still simple enough.